app/models/crosswalk_issue.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class CrosswalkIssue < ApplicationRecord
+  belongs_to :crosswalk
+  belongs_to :ipeds_hd
+  belongs_to :weam
+
+  # rubocop:disable Metrics/MethodLength
+  def self.rebuild
+    sql = <<-SQL
+      INSERT INTO crosswalk_issues (
+        weam_id,
+        crosswalk_id,
+        ipeds_hd_id
+      )
+      SELECT
+        weams.id,
+        crosswalks.id,
+        ipeds_hds.id
+      FROM weams
+        LEFT OUTER JOIN ipeds_hds ON  weams.cross = ipeds_hds.cross
+          OR weams.ope = ipeds_hds.ope
+        LEFT OUTER JOIN crosswalks ON weams.facility_code = crosswalks.facility_code
+      WHERE
+        (institution_of_higher_learning_indicator = true OR non_college_degree_indicator = true)
+        AND NOT(
+          weams.cross IS NOT NULL
+          AND ipeds_hds.cross IS NOT NULL
+          AND crosswalks.cross IS NOT NULL
+          AND weams.cross = ipeds_hds.cross
+          AND weams.cross = crosswalks.cross
+        )
+        AND NOT(
+          weams.cross IS NULL
+          AND weams.ope IS NULL
+          AND ipeds_hds.ope IS NULL
```

Comment:
if `weams.cross` is `null` and `weams.ope` is `null` then `ipeds_hds.ope` will be `null` 100% of the time since the `JOIN` would have failed.  Thus, you don't need this line.

Reply:
Yes, the join would have failed, but because it's an outer join, the result would still include those `weams` rows that failed to join unless we exclude them.

Reply:
Right.  I'm only talking about line 36.  Line 34 and 35 take care of the `weams` rows.
It's a nitpick: I'm just saying that (I think) you will get the same results with or without line 36, because you will never have a result where `weams.cross` is `null` and `weams.ope` is `null` but `ipeds_hds.ope` is present. So you don't need to check for the value of `ipeds_hds.ope`.  Again, not a big deal.